src/compose/ComposeMenu.js

```diff
@@ -4,24 +4,17 @@ import type { ComponentType } from 'react';
 import { Platform, View, Alert, Linking } from 'react-native';
 import type { DocumentPickerResponse } from 'react-native-document-picker';
 import { launchCamera, launchImageLibrary } from 'react-native-image-picker';
-
```

Comment:
Please keep the blank line here; it separates third-party imports from imports of our own code.